unity3d/Assets/Swrve/SwrveSDK/Messaging/SwrveBaseCampaign.cs

```diff
@@ -65,6 +72,42 @@ public class SwrveCampaign
             this.State.Next = value;
         }
     }
+     
+    /// <summary>
+    /// Get the status of the campaign.
+    /// </summary>
+    /// <returns>
+    /// Status of the campaign.
+    /// </returns>
+    public SwrveCampaignState.Status Status {
+        get {
+            return this.State.CurStatus;
+        }
+        set {
+            this.State.CurStatus = value;
+        }
+    }
+        
+    /**
+     * Used internally to identify campaigns that have been marked as MessageCenter campaigns on the dashboard.
+     *
+     * @return true if the campaign is an MessageCenter campaign.
+     */
+    public bool IsMessageCenter() {
```

Comment:
Although it looks like it could be available for IAM only for the time being, but requires tests then